src/ontology/mondo-ingest.Makefile

```diff
@@ -199,13 +217,23 @@ $(REPORTDIR)/%_mapping_status.tsv $(REPORTDIR)/%_unmapped_terms.tsv: $(REPORTDIR
 	--outpath-simple $(REPORTDIR)/$*_unmapped_terms.tsv \
 	--outpath-full $(REPORTDIR)/$*_mapping_status.tsv
 
-components/%-unmapped.owl: components/%.owl reports/%_unmapped_terms.tsv
+# TODO: Issue where Makefile $(COMPONENTSDIR)/% is overriding this goal
+#   Temporarily leaving these commented out until issue resolved.
+#   TODO: docs: If we can't restore $(COMPONENTSDIR)/%-unmapped.owl, need to update workflows.md and --help text at bottom of makefile to reflect goal name change.
+#$(COMPONENTSDIR)/%-unmapped.owl: $(COMPONENTSDIR)/%.owl reports/%_unmapped_terms.tsv
+#	cut -f 1 reports/$*_unmapped_terms.tsv | tail -n +2 > reports/$*_unmapped_terms.txt
+#	$(ROBOT) filter -i components/$*.owl -T reports/$*_unmapped_terms.txt -o $@
+#	rm reports/$*_unmapped_terms.txt
+#.PHONY: recreate-unmapped-components
+#recreate-unmapped-components: $(patsubst %, $(COMPONENTSDIR)/%-unmapped.owl, $(ALL_COMPONENT_IDS))
+.PHONY: %-unmapped.owl
+%-unmapped.owl: $(COMPONENTSDIR)/%.owl reports/%_unmapped_terms.tsv
 	cut -f 1 reports/$*_unmapped_terms.tsv | tail -n +2 > reports/$*_unmapped_terms.txt
-	$(ROBOT) filter -i components/$*.owl -T reports/$*_unmapped_terms.txt -o $@
+	$(ROBOT) filter -i components/$*.owl -T reports/$*_unmapped_terms.txt -o components/$@
 	rm reports/$*_unmapped_terms.txt
 
 .PHONY: recreate-unmapped-components
-recreate-unmapped-components: $(patsubst %, components/%-unmapped.owl, $(ALL_COMPONENT_IDS))
+recreate-unmapped-components: $(patsubst %, %-unmapped.owl, $(ALL_COMPONENT_IDS))
```

Comment:
I dount understand why specifically the introduction of GARD should make that change necessary..

Reply:
This is not GARD-specific. If you want, I can add this change as a separate PR or commit and then `rebase` those changes into this branch, like I did with [this commit](https://github.com/monarch-initiative/mondo-ingest/commit/01b3bbc66861b9c128c7639a8fe58b4bf7638207) while addressing other, more general issues I noticed while working on this PR.

As for why this line changed, it's because the `components/%-unmapped.owl` goal was renamed to `%-unmapped.owl` (see: [this thread](https://github.com/monarch-initiative/mondo-ingest/pull/299#discussion_r1186924007)).